sasslint warnings in various scss files

Assigning to myself, not sure why but we are getting this suddenly on master:

```
Running "sasslint:sass" (sasslint) task

app/styles/_base.scss
    3:2   warning  Pseudo-elements must start with double colons    pseudo-element
    4:2   warning  Pseudo-elements must start with double colons    pseudo-element
  197:9   warning  Attribute values should be surrounded by quotes  attribute-quotes
  209:21  warning  Attribute values should be surrounded by quotes  attribute-quotes
  209:26  warning  Pseudo-elements must start with double colons    pseudo-element
  210:21  warning  Attribute values should be surrounded by quotes  attribute-quotes

app/styles/modules/_avatar.scss
  227:14  warning  Attribute values should be surrounded by quotes  attribute-quotes
  231:14  warning  Attribute values should be surrounded by quotes  attribute-quotes

app/styles/modules/_cropper.scss
  25:6  warning  Pseudo-elements must start with double colons  pseudo-element
  26:6  warning  Pseudo-elements must start with double colons  pseudo-element
  37:6  warning  Pseudo-elements must start with double colons  pseudo-element
  41:6  warning  Pseudo-elements must start with double colons  pseudo-element

app/styles/modules/_custom-rows.scss
  188:16  warning  Attribute values should be surrounded by quotes  attribute-quotes

app/styles/modules/_input-row.scss
  30:12  warning  Attribute values should be surrounded by quotes  attribute-quotes
  31:12  warning  Attribute values should be surrounded by quotes  attribute-quotes
  38:12  warning  Attribute values should be surrounded by quotes  attribute-quotes

app/styles/modules/_password-row.scss
   10:14  warning  Attribute values should be surrounded by quotes  attribute-quotes
   14:14  warning  Attribute values should be surrounded by quotes  attribute-quotes
   21:14  warning  Attribute values should be surrounded by quotes  attribute-quotes
   25:14  warning  Attribute values should be surrounded by quotes  attribute-quotes
   69:14  warning  Attribute values should be surrounded by quotes  attribute-quotes
   74:14  warning  Attribute values should be surrounded by quotes  attribute-quotes
   85:16  warning  Attribute values should be surrounded by quotes  attribute-quotes
   98:14  warning  Attribute values should be surrounded by quotes  attribute-quotes
  102:14  warning  Attribute values should be surrounded by quotes  attribute-quotes
  109:14  warning  Attribute values should be surrounded by quotes  attribute-quotes
  113:14  warning  Attribute values should be surrounded by quotes  attribute-quotes

app/styles/modules/_settings.scss
   43:14  warning  Attribute values should be surrounded by quotes  attribute-quotes
   47:14  warning  Attribute values should be surrounded by quotes  attribute-quotes
   56:16  warning  Attribute values should be surrounded by quotes  attribute-quotes
   61:16  warning  Attribute values should be surrounded by quotes  attribute-quotes
   72:16  warning  Attribute values should be surrounded by quotes  attribute-quotes
   77:16  warning  Attribute values should be surrounded by quotes  attribute-quotes
   89:14  warning  Attribute values should be surrounded by quotes  attribute-quotes
   93:14  warning  Attribute values should be surrounded by quotes  attribute-quotes
  180:16  warning  Attribute values should be surrounded by quotes  attribute-quotes
  184:16  warning  Attribute values should be surrounded by quotes  attribute-quotes
  347:14  warning  Attribute values should be surrounded by quotes  attribute-quotes
  351:14  warning  Attribute values should be surrounded by quotes  attribute-quotes
  358:16  warning  Attribute values should be surrounded by quotes  attribute-quotes
  362:16  warning  Attribute values should be surrounded by quotes  attribute-quotes
  370:16  warning  Attribute values should be surrounded by quotes  attribute-quotes
  374:16  warning  Attribute values should be surrounded by quotes  attribute-quotes

app/styles/modules/_tooltip.scss
  10:12  warning  Attribute values should be surrounded by quotes  attribute-quotes
  14:12  warning  Attribute values should be surrounded by quotes  attribute-quotes
  20:4   warning  Pseudo-elements must start with double colons    pseudo-element
  34:14  warning  Attribute values should be surrounded by quotes  attribute-quotes
  38:14  warning  Attribute values should be surrounded by quotes  attribute-quotes
  46:6   warning  Pseudo-elements must start with double colons    pseudo-element
  79:31  warning  Pseudo-elements must start with double colons    pseudo-element
  81:15  warning  Pseudo-elements must start with double colons    pseudo-element

✖ 51 problems (0 errors, 51 warnings)
```

--- a/app/styles/_base.scss
+++ b/app/styles/_base.scss
@@ -1,7 +1,7 @@
 // Border box all the things by default
 *,
-*:before,
-*:after {
+*::before,
+*::after {
   box-sizing: border-box;
 }
 
@@ -194,7 +194,7 @@ section p {
 //
 // hrefs are only visible from the app, when the TOS/PP agreements
 // are opened directly, the links display/act normally.
-a[href^=http].visible-url {
+a[href^='http'].visible-url {
   // using text-decoration: underline underlines the ::after
   // section as well, with no way to remove it.
   // So, add a border to the entire element, then hide
@@ -206,8 +206,7 @@ a[href^=http].visible-url {
   text-decoration: none;
 }
 
-a[data-visible-url^=http]:after,
-a[data-visible-url^=http]::after {
+a[data-visible-url^='http']::after {
   border-bottom: 1px solid $content-background-color;
   content: ' (' attr(href) ') ';
   word-break: break-all;

--- a/app/styles/modules/_avatar.scss
+++ b/app/styles/modules/_avatar.scss
@@ -224,11 +224,11 @@ section.avatar-panel {
   .button-row {
     margin-bottom: 0;
 
-    html[dir=ltr] & {
+    html[dir='ltr'] & {
       text-align: left;
     }
 
-    html[dir=rtl] & {
+    html[dir='rtl'] & {
       text-align: right;
     }
   }

--- a/app/styles/modules/_cropper.scss
+++ b/app/styles/modules/_cropper.scss
@@ -22,8 +22,8 @@
     width: $avatar-size;
     z-index: 1;
 
-    &:before,
-    &:after {
+    &::before,
+    &::after {
       background-color: rgba($cropper-background-color, 0.5);
       content: '';
       display: block;
@@ -34,11 +34,11 @@
       z-index: 1;
     }
 
-    &:before {
+    &::before {
       left: -(420 - $avatar-size) / 2;
     }
 
-    &:after {
+    &::after {
       right: -(420 - $avatar-size) / 2;
     }
   }

--- a/app/styles/modules/_custom-rows.scss
+++ b/app/styles/modules/_custom-rows.scss
@@ -182,7 +182,7 @@ ul.links {
   color: $input-row-focus-border-color;
 }
 
-.password[type=text]:focus ~ .show-password-label {
+.password[type='text']:focus ~ .show-password-label {
   background-color: $input-row-focus-border-color;
   color: $show-password-focus-label-color;
 }

--- a/app/styles/modules/_input-row.scss
+++ b/app/styles/modules/_input-row.scss
@@ -27,15 +27,15 @@
                 box-shadow $short-transition;
 
     // disable arrows on HTML5 number inputs
-    &[type=number]::-webkit-inner-spin-button,
-    &[type=number]::-webkit-outer-spin-button {
+    &[type='number']::-webkit-inner-spin-button,
+    &[type='number']::-webkit-outer-spin-button {
       -webkit-appearance: none;
       -moz-appearance: none;
       appearance: none;
       margin: 0;
     }
 
-    &[type=number] {
+    &[type='number'] {
       -moz-appearance: textfield;
     }
 

--- a/app/styles/modules/_password-row.scss
+++ b/app/styles/modules/_password-row.scss
@@ -7,22 +7,22 @@
 
   &.input-row .password {
 
-    html[dir=ltr] & {
+    html[dir='ltr'] & {
       padding-right: 75px;
     }
 
-    html[dir=rtl] & {
+    html[dir='rtl'] & {
       padding-left: 75px;
     }
   }
 
   .input-help {
 
-    html[dir=ltr] & {
+    html[dir='ltr'] & {
       text-align: left;
     }
 
-    html[dir=rtl] & {
+    html[dir='rtl'] & {
       text-align: right;
     }
   }
@@ -66,12 +66,12 @@
     user-select: none;
     z-index: 3;
 
-    html[dir=ltr] & {
+    html[dir='ltr'] & {
       border-left: 1px solid $input-row-border-color;
       right: 1px;
     }
 
-    html[dir=rtl] & {
+    html[dir='rtl'] & {
       border-right: 1px solid $input-row-border-color;
       left: 1px;
     }
@@ -82,7 +82,7 @@
       height: 43px;
       line-height: 42px;
 
-      html[dir=rtl] & {
+      html[dir='rtl'] & {
         border-radius: $small-border-radius 0 0 $small-border-radius;
       }
     }
@@ -95,22 +95,22 @@
   }
 
   .password:focus ~ .show-password-label {
-    html[dir=ltr] & {
+    html[dir='ltr'] & {
       border-left-color: $input-row-focus-border-color;
     }
 
-    html[dir=rtl] & {
+    html[dir='rtl'] & {
       border-right-color: $input-row-focus-border-color;
     }
   }
 
   .password.invalid ~ .show-password-label {
 
-    html[dir=ltr] & {
+    html[dir='ltr'] & {
       border-left: 1px solid $error-background-color !important;
     }
 
-    html[dir=rtl] & {
+    html[dir='rtl'] & {
       border-right: 1px solid $error-background-color !important;
     }
   }

--- a/app/styles/modules/_settings.scss
+++ b/app/styles/modules/_settings.scss
@@ -40,11 +40,11 @@ body.settings #main-content.card {
     background-repeat: no-repeat;
     margin: 0;
 
-    html[dir=ltr] & {
+    html[dir='ltr'] & {
       float: left;
     }
 
-    html[dir=rtl] & {
+    html[dir='rtl'] & {
       float: right;
     }
 
@@ -53,12 +53,12 @@ body.settings #main-content.card {
       height: 64px;
       line-height: 63px;
 
-      html[dir=ltr] & {
+      html[dir='ltr'] & {
         background-position: left 11px;
         padding-left: 48px;
       }
 
-      html[dir=rtl] & {
+      html[dir='rtl'] & {
         background-position: right 11px;
         padding-right: 48px;
       }
@@ -69,12 +69,12 @@ body.settings #main-content.card {
       height: 48px;
       line-height: 50px;
 
-      html[dir=ltr] & {
+      html[dir='ltr'] & {
         background-position: left 8px;
         padding-left: 36px;
       }
 
-      html[dir=rtl] & {
+      html[dir='rtl'] & {
         background-position: right 8px;
         padding-right: 36px;
       }
@@ -86,11 +86,11 @@ body.settings #main-content.card {
     display: block;
     height: 30px;
 
-    html[dir=ltr] & {
+    html[dir='ltr'] & {
       float: right;
     }
 
-    html[dir=rtl] & {
+    html[dir='rtl'] & {
       float: left;
     }
 
@@ -177,11 +177,11 @@ body.settings #main-content.card {
 
     &:last-child {
 
-      html[dir=ltr] & {
+      html[dir='ltr'] & {
         float: right;
       }
 
-      html[dir=rtl] & {
+      html[dir='rtl'] & {
         float: left;
       }
 
@@ -344,22 +344,22 @@ body.settings #main-content.card {
     margin-top: 2px;
     overflow: hidden;
 
-    html[dir=ltr] & {
+    html[dir='ltr'] & {
       float: left;
     }
 
-    html[dir=rtl] & {
+    html[dir='rtl'] & {
       float: right;
     }
 
     @include respond-to('big') {
       width: 350px;
 
-      html[dir=ltr] & {
+      html[dir='ltr'] & {
         margin-right: 10px;
       }
 
-      html[dir=rtl] & {
+      html[dir='rtl'] & {
         margin-left: 10px;
       }
     }
@@ -367,11 +367,11 @@ body.settings #main-content.card {
     @include respond-to('small') {
       width: 70%;
 
-      html[dir=ltr] & {
+      html[dir='ltr'] & {
         margin-right: 1%;
       }
 
-      html[dir=rtl] & {
+      html[dir='rtl'] & {
         margin-left: 1%;
       }
     }

--- a/app/styles/modules/_tooltip.scss
+++ b/app/styles/modules/_tooltip.scss
@@ -7,17 +7,16 @@
   top: -32px;
   z-index: 5;
 
-  html[dir=ltr] & {
+  html[dir='ltr'] & {
     left: 3px;
   }
 
-  html[dir=rtl] & {
+  html[dir='rtl'] & {
     right: 3px;
   }
 
 
   /* tooltip caret */
-  &:before,
   &::before {
     background: $error-background-color;
     bottom: -8px;
@@ -31,11 +30,11 @@
     // The z-index must be -1 or else the caret is displayed on top of the tooltip text
     z-index: -1;
 
-    html[dir=ltr] & {
+    html[dir='ltr'] & {
       left: 12px;
     }
 
-    html[dir=rtl] & {
+    html[dir='rtl'] & {
       right: 12px;
     }
   }
@@ -43,7 +42,6 @@
   &.tooltip-suggest {
     background: $button-background-color;
 
-    &:before,
     &::before {
       background-color: $button-background-color;
     }
@@ -76,9 +74,7 @@
  * these are caret's attached to tooltips that hang below the
  * input element. The tooltip's top is set via JS.
  */
-html[dir='rtl'] .tooltip-below:before,
 html[dir='rtl'] .tooltip-below::before,
-.tooltip-below:before,
 .tooltip-below::before {
   top: -7px;
 }

